src/HttpKernel/Bundle/ContaoModuleAutoloadBundle.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\HttpKernel\Bundle;
+
+use Mmoreram\SymfonyBundleDependencies\DependentBundleInterface;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+/**
+ * Automatically register all legacy modules as dependencies.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+final class ContaoModuleAutoloadBundle extends Bundle implements DependentBundleInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public static function getBundleDependencies(KernelInterface $kernel)
+    {
+        $kernelDir  = $kernel->getRootDir();
+        $bundles    = [];
+
+        foreach (static::getContaoModules(dirname($kernelDir)) as $module) {
```

Comment:
Does this mean that every module using this class has **every** non-skippable module as dependency?